src/lib/core/common-behaviors/common-module.ts

```diff
@@ -60,7 +60,9 @@ export class MdCommonModule {
       testElement.classList.add('mat-theme-loaded-marker');
       this._document.body.appendChild(testElement);
 
-      if (getComputedStyle(testElement).display !== 'none') {
+      const computedStyle = getComputedStyle(testElement);
+
+      if (computedStyle && computedStyle.display !== 'none') {
```

Comment:
Add a comment that it can potentially be null?

Reply:
Done. Please have another look